doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -129,12 +129,16 @@ To enable one of the reactions to the last fabric removal, set the corresponding
 * :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` - Remove all saved network credentials and start Bluetooth LE advertising.
   The device will remove all saved network credentials, keep application-specific non-volatile data, and start advertising Bluetooth LE Matter service.
   After that, it will be ready for commissioning to Matter over Bluetooth LE.
+  This option is not available in the :ref:`matter_template_sample` sample.
 * :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_REBOOT` - Remove all saved network credentials and reboot the device.
   This option is selected by default.
 
   When the :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option is also set to ``y``, the device will also remove all non-volatile data stored on the device, including application-specific entries.
   This means the device is restored to the factory settings.
 
+To create a delay between  the chosen reaction and the last fabric being removed, set the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_SCHEDULE_TIME` Kconfig option to a specific time in milliseconds.
+By default this Kconfig option is set to 500 milliseconds.
```

Comment:
Minor: Isn't linking to kconfig option enough? I mean, the user can click on the link and see the default value. And when the value is directly given in the text, we will have to update docs after modifying the Kconfig value.

Reply:
Since we have a problem with linking, I think we should leave it as it is.  Shouldn't we?

Reply:
If the fix for links will not come shortly I'm ok with leaving it as it is.